sonar-html-plugin/src/main/java/org/sonar/plugins/html/api/HtmlConstants.java

```diff
@@ -141,13 +161,25 @@ public class HtmlConstants {
     "title",
     "tr",
     "track",
+    "tt", // deprecated
     "u",
     "ul",
     "var",
     "video",
-    "wbr"
+    "wbr",
+    "xmp" // deprecated
   );
 
+  // computed from https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/src/util/isNonInteractiveElement.js
+  public static final Set<String> NON_INTERACTIVE_ELEMENTS = Set.of("abbr", "address", "article", "aside", "blockquote", "br", "caption", "code", "dd", "del", "details", "dfn",
+    "dialog", "dir", "dl", "dt", "em", "fieldset", "figcaption", "figure", "footer", "form", "h1", "h2", "h3", "h4", "h5", "h6", "hr", "html", "iframe", "img", "ins", "label",
+    "legend", "li", "main", "mark", "marquee", "menu", "meter", "nav", "ol", "optgroup", "output", "p", "pre", "progress", "ruby", "strong", "sub", "sup", "table", "tbody",
+    "tfoot", "thead", "time", "ul");
+  // computed as https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/src/util/isInteractiveRole.js
+  public static final Set<String> INTERACTIVE_ROLES = Set.of("button", "checkbox", "columnheader", "combobox", "grid", "gridcell", "link", "listbox", "menu", "menubar", "menuitem",
```

Comment:
Could you add another set for noninteractive roles?

Reply:
similar to https://github.com/SonarSource/sonar-html/pull/273/files#r1495810809